assets/sass/protocol/base/elements/_forms.scss

```diff
@@ -55,7 +55,7 @@ button,
 [type='button'],
 [type='reset'],
 [type='submit'] {
-    appearance: button;
+    appearance: auto;
```

Comment:
`button` value is deprecated: https://drafts.csswg.org/css-ui-4/#valdef-appearance-button

we may need to disable stylelint for this rule, as `auto` support might not go back far enough: https://caniuse.com/?search=appearance%3A%20auto

Reply:
(I may need to check properly, but doesn't the build take care of legacy output based on the caniuse data?)